app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -96,7 +96,7 @@ class ProfileFragment : Fragment() {
                 val bundle = Bundle()
                 bundle.putString(EMAIL, emailSettings)
                 fragment.arguments = bundle
-                activity?.supportFragmentManager?.beginTransaction()?.replace(R.id.frameContainer, fragment)?.addToBackStack(null)?.commit()
+                activity?.supportFragmentManager?.beginTransaction()?.replace(R.id.rootLayout, fragment)?.addToBackStack(null)?.commit()
```

Comment:
why is this changed?

Reply:
Settings Fragment now covers the whole screen